### PR TITLE
Update jQuery.json

### LINKED ISSUE #9303: Get Geolocation Data: jQuery selector with single quote won't pass
Challenge [Get Geolocation Data](https://www.freecodecamp.com/challenges/get-geolocation-data#?solution=%0Afccss%0A%20%20%2F%2F%20Only%20change%20code%20below%20this%20line.%0Aif%20(navigator.geolocation)%20%7B%0A%20%20navigator.geolocation.getCurrentPosition(function(position)%20%7B%0A%20%20%20%20%24(%22%23data%22).html(%22latitude%3A%20%22%20%2B%20position.coords.latitude%20%2B%20%22%3Cbr%3Elongitude%3A%20%22%20%2B%20position.coords.longitude)%3B%0A%20%20%7D)%3B%0A%7D%0A%0A%20%20%2F%2F%20Only%20change%20code%20above%20this%20line.%0Afcces%0A%3Cdiv%20id%20%3D%20%22data%22%3E%0A%20%20%3Ch4%3EYou%20are%20here%3A%3C%2Fh4%3E%0A%20%20%0A%3C%2Fdiv%3E%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html

<script>
  // Only change code below this line.
if (navigator.geolocation) {
  navigator.geolocation.getCurrentPosition(function(position) {
    $('#data').html("latitude: " + position.coords.latitude + "<br>longitude: " + position.coords.longitude);
  });
}

  // Only change code above this line.
</script>
<div id = "data">
  <h4>You are here:</h4>
  
</div>

```

The code is 99% similar with the given example code, except the single quote of jQuery selector.

If I change `$('#data')` to `$("#data")`, the last test case **You should display the users' position within the data div element** can be passed.

Single quote for `string` can be passed, this seems only happen for jQuery selectors.

### PR DESCRIPTION
**Pull request by sthodup1**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [ ] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [ ] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [ ] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #9303
#### Description

Tells people to use double quotes in selectors in order to pass the tests and to follow jQuery style guidelines.

### CODE REVIEW

**Comment by sthodup1** [comment]
@BerkeleyTrue Can you help me figure out why this PR failed the check?

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 1]
```diff
@@ -81,6 +81,7 @@
         "For example, let's make all of your <code>button</code> elements bounce. Just add this code inside your document ready function:",
         "<code>$(\"button\").addClass(\"animated bounce\");</code>",
         "Note that we've already included both the jQuery library and the Animate.css library in the background so that you can use them in the editor. So you are using jQuery to apply the Animate.css <code>bounce</code> class to your <code>button</code> elements."
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/jquery.json]: add a `,` at the end of line.

**Comment by raisedadead** [comment]
You can always test lining errors as mentioned in the Contributing Guidelines with `npm run test-challenges`

File: seed/challenges/01-front-end-development-certification/jquery.json (hunk 1 above)
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/jquery.json]: This is causing the CI failure.

**Comment by raisedadead** [comment]
@sthodup1 thanks for updating, please squash your commit to one, update the message to a litlle bit more descriptive and make the PR title a bit descriptive as well. 

You can find the steps in the Contributing guide

**Comment by sthodup1** [comment]
@raisedadead Thanks for the help! I'll work on squashing it now.

**Pull request closed without merging**